### DISCUSSION TITLE
Automatically replace non compatible pool plots with compatible ones

### DISCUSSION THREAD

**Discussion by tomskipomski** [swar/Swar-Chia-Plot-Manager#653] [Ideas]
Hi, 

would love to have an option to automatically delete a non compatible pool plot with a newly generated compatible one on the fly.

So say for example all my HDDs are full of noncompatible pool plots. If pools are out I need to replot and replace them with compatible pool plots. And instead of keeping up to manually deleting the plots the plot manager could replace the old plot with the new one on the fly after phase 4.

**Comment by Rayrayray87** [comment]
I would pay money for this feature, it would save so much time.

**Reply by MinerGreggy** [reply]
i had that idea also... #648
